spark-indexer/src/test/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtKBEntryConverterUtilTest.java

```diff
@@ -253,4 +251,28 @@ void canNotAddExperimentalWithoutExperimentalAndNotReviewed() {
                 UniProtEntryConverterUtil.canAddExperimental(
                         true, "Valid Text", false, Collections.emptySet()));
     }
+
+    @Test
+    void canAddSuggestion() {
+        UniProtDocument doc = new UniProtDocument();
+
+        Collection<String> values = Set.of("value1", "value2");
+        UniProtEntryConverterUtil.populateSuggestions(values, doc);
+
+        assertNotNull(doc.suggests);
+        assertEquals(2, doc.suggests.size());
+        assertEquals(values, doc.suggests);
+    }
+
+    @Test
+    void canAddSuggestionFilterSmallValues() {
+        UniProtDocument doc = new UniProtDocument();
+
+        Collection<String> values = Set.of("val", "value2");
+        UniProtEntryConverterUtil.populateSuggestions(values, doc);
+
+        assertNotNull(doc.suggests);
+        assertEquals(1, doc.suggests.size());
+        assertTrue(doc.suggests.contains("value2"));
+    }
 }
```

Comment:
What if null or empty is passed as values? Could you please cover those cases?

Reply:
thank you, I added extra Unit tests